src/renderer/utils/Directories.ts

```diff
@@ -29,28 +29,28 @@ export class Directories {
     }
 
     static temp(): string {
-        return path.join(settings.get('mainSettings.msfsPackagePath') as string, `flybywire_current_install_${(Math.random() * 1000).toFixed(0)}`);
```

Comment:
See this function is creating a unique temporary folder for each ongoing installation. You could change the first part of the path to use a different variable that can be set in settings. Please consider it to be an optional setting, meaning if not enabled, it should dynamically change with the community directory. See https://github.com/flybywiresim/installer/pull/230/files for reference but take into account we switched to toggles instead of checkboxes

Reply:
Ah, so I'll create a checkbox under it to toggle a dedicated temporary folder

Reply:
I think I've done it, let me know if I haven't